ParameterDataApi.yaml

```diff
@@ -88,122 +144,79 @@ components:
           description: The success message provided back to the client
       required:
         - message
-        
-    ColumnDefinition:
-      type: array
-      items:
-        $ref: '#/components/schemas/ColumnElementDefinition'
-          
-    ColumnElementDefinition:
-      type: object
-      description: The column elements      
-      properties:
-        
-        name: 
-          description: The name of the column
-          type: string
-          pattern: '^[a-zA-Z0-9]{2,}'
-          
-        dataType:
-          $ref: '#/components/schemas/DataType'
-                   
-      required:
-        - name
-        - dataType
 
-    DataDefinition:
-      type: array
-      items: 
-        $ref: '#/components/schemas/DataElement'
-        
-    DataElement:
+    DataSample:
       type: object
+      description: A definition of a set of parameter data having a single discrete timestamp
       properties:
+      
         timestamp:
```

Comment:
Unsure if timestamp or timeOfApplicability is more appropriate here, given previous discussions and type of data being passed